src/SprykerEco/Zed/Adyen/Business/Hook/Mapper/MakePayment/KlarnaInvoiceMapper.php

```diff
@@ -77,10 +73,10 @@ function (ItemTransfer $item) {
                     ->setId($item->getSku())
                     ->setDescription($item->getName())
                     ->setQuantity($item->getQuantity())
-                    ->setTaxAmount($item->getSumTaxAmount())
-                    ->setTaxPercentage((int)$item->getTaxRate())
-                    ->setAmountExcludingTax($item->getSumPrice() - $item->getSumTaxAmount())
-                    ->setAmountIncludingTax($item->getSumPrice());
+                    ->setTaxAmount($item->getSumTaxAmountFullAggregation())
+                    ->setTaxPercentage((int)$item->getTaxRate() * static::REQUEST_TAX_RATE_MULTIPLIER)
```

Comment:
I would like to have it in separate method as requirements can be changed in future and it will be easer to extend it on project level.

Reply:
Done